aspnetcore/fundamentals/logging/index.md

```diff
@@ -919,15 +919,119 @@ On Linux, this provider writes logs to */var/log/message*.
 logging.AddDebug();
 ```
 
-### EventSource provider
+### Event Source provider
 
-For apps that target ASP.NET Core 1.1.0 or later, the [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing. On Windows, it uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803). The provider is cross-platform, but there are no event collection and display tools yet for Linux or macOS.
+The [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package writes to an Event Source cross-platform with the name `Microsoft-Extensions-Logging`. On Windows, the provider uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803).
 
 ```csharp
 logging.AddEventSourceLogger();
 ```
 
-A good way to collect and view logs is to use the [PerfView utility](https://github.com/Microsoft/perfview). There are other tools for viewing ETW logs, but PerfView provides the best experience for working with the ETW events emitted by ASP.NET Core.
+The Event Source provider is added automatically when `CreateDefaultBuilder` is called to build the host.
+
+::: moniker range=">= aspnetcore-3.0"
+
+#### dotnet trace tooling
+
+The [dotnet-trace](/dotnet/core/diagnostics/dotnet-trace) tool allows you to consume <xref:Microsoft.Extensions.Logging.EventSource> provider data using a <xref:Microsoft.Extensions.Logging.EventSource.LoggingEventSource>.
+
+Install the dotnet trace tooling by running the following command in a command shell:
+
+```dotnetcli
+dotnet tool install --global dotnet-trace
+```
+
+Use the dotnet trace tooling to collect a trace from an app:
+
+1. Run the app with the `dotnet run` command (Development environment).
```

Comment:
Run the app with the `dotnet run` command.

Reply:
Glad you pinged on this one. I forgot that I wanted to add that the Event Source Provider isn't added if `CreateDefaultBuilder` isn't called, so it needs a line and a cross-link. Yes, we don't need to mention the environment. That was leftover from when I thought it was relevant.